src/webgpu/web-platform/reftests/present_to_canvas.ts

```diff
@@ -0,0 +1,45 @@
+import { GPURefTest } from './gpu_ref_test.js';
+
+async function PresentToCanvasTest(): Promise<void> {
+  const t = new GPURefTest();
+  await t.init();
+  if (typeof document === 'undefined') {
+    // Skip if there is no document (Workers, Node)
+    console.log('DOM is not available to create canvas element');
+    return;
+  }
```

Comment:
Not needed because this file is only ever included from the html file (so the DOM is always available)